File: asv_control/src/testnode.cpp
#include "rclcpp/rclcpp.hpp"

class TestNode : public rclcpp::Node 
{
public:
    TestNode() : Node("test")
    {
    	RCLCPP_INFO(this->get_logger(), "Test Node cpp has been started.");
    }

private:
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<TestNode>();
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}

File: asv_control/scripts/testnode.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node


class TestNode(Node):
    def __init__(self):
        super().__init__("test")
        self.get_logger().info("Test Node py has been started")


def main(args=None):
    rclpy.init(args=args)
    node = TestNode()
    rclpy.spin(node)
    rclpy.shutdown()


if __name__ == "__main__":
    main()